### PR TITLE
Fix lint errors

### PR DESCRIPTION
**Pull request by kosssi**

### CODE REVIEW

**Comment by jacquarg** [comment]
:+1:

File: www-src/test/unit/replicator/design_documents.coffee [hunk 1]
```diff
@@ -13,10 +13,10 @@ module.exports = describe 'DesignDocuments Test', ->
         designDocs.createOrUpdateAllDesign (error, responses) ->
             async.series [
                 (next) -> cozyDB.allDocs {}, (error, response) ->
-                    assert.equal 8, response.total_rows
+                    8.should.be.equal response.total_rows
```
Comment (nono) [inline www-src/test/unit/replicator/design_documents.coffee]: It reads better in the reverse order:

```coffee
response.total_rows should.equal 8
```

**Comment by nono** [comment]
The build on travis is red. Was it expected?

**Comment by kosssi** [comment]
@nono we merge in another branch so it's not a problem for now ;)

**Pull request merged**